### DISCUSSION TITLE
How to Bulk Update Records?

### DISCUSSION THREAD

**Discussion by c2006878** [nocodb/nocodb#8978] [Q&A]
I couldn't find the button for "Update Selected Rows" as shown in the following image
![image](https://github.com/nocodb/nocodb/assets/58558960/0c220897-8aaa-4761-aea3-a92bed505375)

**Comment by o1lab** [comment]
<img width="345" alt="image" src="https://github.com/nocodb/nocodb/assets/5435402/e54a8ef0-c501-4585-9403-c282ecbfa0cb">

Its only available in the cloud or enterprise edition. Please watch out for the cloud icon.

**Reply by benhill** [reply]
I ran into this today. This is pretty lame.  This is a mundane feature - so odd you decided to put it behind a paywall.
